improver/cli/nowcast_accumulate.py

```diff
@@ -31,26 +31,55 @@
 # POSSIBILITY OF SUCH DAMAGE.
 """Script to accumulate input data given advection velocity fields."""
 
+from typing import Callable, List
+
 from improver import cli
 
 # The accumulation frequency in minutes.
 ACCUMULATION_FIDELITY = 1
 
+
+def name_constraint(names: List[str]) -> Callable:
+    """
+    Generates a callable constraint for matching cube names.
+
+    The callable constraint will realise the data of those cubes matching the
+    constraint.
+    cube names
+
+    Args:
+        name:
+            List of cube names to constrain our cubes.
+
+    Returns:
+        A callable which when called, returns True or False for the provided cube,
+        depending on whether it matches the names provided.  These matching cube
+        will also have its data realised by the callable.
+    """
+
+    def constraint(cube):
+        ret = False
+        if cube.name() in names:
+            ret = True
+            _ = cube.data
```

Comment:
Does this need to be saved to `_`? Or is simply loading the data using `cube.data` enough?

Reply:
It's not needed.  A convention followed in AVD.  I'm not actually sure why.
I'll change.

Reply:
I'm guessing it's just about making it explicit that we are ignoring the return...